script/urlChecker.ts

```diff
@@ -77,19 +85,23 @@ const getUrl = (book: Book) => useUnversionedUrls
   : (treeSection: LinkedArchiveTreeSection) => getBookPageUrlAndParams(book, treeSection).url;
 
 async function checkUrls() {
+  const archiveLoader = createArchiveLoader(`${archiveUrl ? archiveUrl : rootUrl}${config.REACT_APP_ARCHIVE_URL}`);
+  const osWebLoader = createOSWebLoader(`${rootUrl}${config.REACT_APP_OS_WEB_API_URL}`);
   const url = assertDefined(rootUrl, 'please define a rootUrl parameter, format: http://host:port');
   const books = await findBooks({
-    archiveUrl,
+    archiveLoader,
     bookId,
     bookVersion,
+    osWebLoader,
     rootUrl: url,
   });
+  const redirects = await prepareRedirects(archiveLoader, osWebLoader);
```

Comment:
idk if there is any point in running this

Reply:
@TomWoodward 
What do you mean? Do I have access to `rex/redirects.json` during CI?

Reply:
`prepareRedirects` doesn't add any information you don't already have, it just renders the book id and page id into a url, you're comparing the pages to urls and comparing the urls, you could just compare the ids